src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -43,6 +46,28 @@ public static double calculateSlideAmount(double currentNum, int targetNum)
     return slide * sign;
   }
 
+  private static HashMap<Color, Sprite> coloredCursors;
+  public static void drawCursor(Graphics g, int x, int y, int w, int h, Color color)
+  {
+    if( null == coloredCursors )
+    {
+      coloredCursors = new HashMap<Color, Sprite>();
+    }
+    if( !coloredCursors.containsKey(color) )
+    {
+      Sprite newCursor = new Sprite(SpriteLibrary.getCursorSprites());
+      newCursor.colorize(UIUtils.defaultMapColors[4], color);
+      coloredCursors.put(color, newCursor);
+    }
+
+    // Draw the arrows around the focused player attribute.
```

Comment:
Is there any reason we can't use this logic for the map cursor?

Reply:
That's the plan. I think I want to reify the cursor and animate it a bit before taking that step though.